src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreams.java

```diff
@@ -25,6 +26,9 @@ static MtAmazonDynamoDbStreams createFromDynamo(AmazonDynamoDB dynamoDb, AmazonD
         }
 
         if (dynamoDb instanceof MtAmazonDynamoDbBySharedTable) {
+            AmazonDynamoDBStreams streams = dynamoDbStreams instanceof CachingAmazonDynamoDbStreams
```

Comment:
Seems like caching would never be turned on here.

Reply:
I also noticed that when prepping for the demo :). Impressive that you caught this just by inspecting the code!